### DISCUSSION TITLE
Problems with LR12 and MIDI2LR.exe not opening

### DISCUSSION THREAD

**Discussion by cutlerphoto** [rsjaffe/MIDI2LR#728] [Problems]
Haven't been using the plugin for a while, now it's completely stopped working. Doesn't load screen displaying profile settings, nothing.

Windows 10. newest version of midi2LR has been uninstalled, reinstalled, etc.

Nothing works.

**Comment by antzpantz** [comment]
Try deleting C:\Users\cutlerphoto\AppData\Roaming\MIDI2LR
and C:\Users\cutlerphoto\AppData\Roaming\Adobe\Lightroom\Modules\MIDI2LR.lrplugin
and
reinstall.

**Comment by cutlerphoto** [comment]
I have tried that after some other searching. Perhaps a Windows update is
preventing the app from loading also? I have firewall disabled and have
tried running as administrator when installing though, not sure if either
mattered.
